client/components/Channel.vue

```diff
@@ -36,6 +36,8 @@ export default {
 	props: {
 		network: Object,
 		channel: Object,
+		active: Boolean,
+		isFiltering: Boolean,
```

Comment:
Is `isFiltering` never used?

Reply:
Yeah, passed down from `NetworkList.vue` with `:is-filtering="true"` so a bit hard to spot with find or grep etc.